# Message header
std_msgs/Header header

# positon in WGS84
float64 latit
float64 longi
float32 altit

float32 velN
float32 velE
float32 velD

float32 atti_pitch
float32 atti_roll
float32 atti_yaw

float32 gyro_pitch
float32 gyro_roll
float32 gyro_yaw

float32 accN
float32 accE
float32 accD

float32[] quat


